download-basket/src/main/java/downloadbasket/data/NormalWayDownloads.java

```diff
@@ -0,0 +1,47 @@
+package downloadbasket.data;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class NormalWayDownloads {
+	private final Logger LOGGER = LogFactory.getLogger(NormalWayDownloads.class);
+	private Set<String> downloads = new HashSet<>();
+
+	/**
+	 * Add normal way download.
+	 * 
+	 * @param download
+	 */
+	public void addDownload(String download) {
+		downloads.add(download);
+	}
+
+	/**
+	 * Cheks if download is cropped with BBOX. Normal way uses BBOX as the
+	 * cropping method. If not, GeoServer Cross-layer filtering is used.
+	 * 
+	 * @param croppingMode
+	 *            the cropping mode
+	 * @param croppingLayer
+	 *            the cropping layer
+	 * @return
+	 */
+	public boolean isBboxCropping(String croppingMode, String croppingLayer) {
+		boolean isNormalWay = false;
```

Comment:
The implementation for this could be a oneliner?

     public boolean isBboxCropping(String croppingMode) {
         return downloads.contains(croppingMode);
    }

You only pass the layer param for logging purposes. Maybe log it outside the method if you really need to?

Reply:
Or you could just remove the class entirely and use Set<String> wherever NormalWayDownloads is used?